funnel/views/helpers.py

```diff
@@ -57,8 +57,13 @@ def mask_email(email):
 
 
 def clear_old_session(response):
-    if 'session' in request.cookies and app.config.get('SESSION_COOKIE_NAME') != 'session':
-        response.set_cookie('session', '', expires=0, httponly=True)
+    if app.config.get('DELETE_COOKIES'):
+        for cookie_name in app.config.get('DELETE_COOKIES'):
```

Comment:
Remove the `if` and do `for cookie_name, domains in app.config.get('DELETE_COOKIES', {}):`